jerry-core/ecma/builtin-objects/ecma-builtin-array-prototype.c

```diff
@@ -850,7 +850,8 @@ ecma_builtin_array_prototype_object_slice (ecma_value_t arg1, /**< start */
 
     if (ext_from_obj_p->u.array.u.hole_count < ECMA_FAST_ARRAY_HOLE_ONE
         && len != 0
-        && start < end)
+        && start < end
+        && end - start > len)
```

Comment:
Correct me if I'm wrong, but both `start` and `end` are in the [0, len] range, and we we know that `start < end` from the condition, which means `end - start` will allways be `<= len`.

So essentially this just disables the fast path entirely.

Reply:
Good point, this might be a wrong fix for the issue.

Reply:
Right, I missed that. I will come up with another solution.